packages/result/src/lib/Option/IOption.ts

```diff
@@ -206,6 +223,23 @@ export interface IOption<T> {
 	 */
 	mapOrElse<U>(defaultValue: () => U, cb: (value: T) => U): U;
 
+	/**
+	 * Maps a `None` to the returned `Option<U>` by applying a function to a contained value, leaving `Some<T>`
+	 * untouched.
+	 * @param cb The predicate.
+	 *
+	 * @example
+	 * ```typescript
+	 * const maybeSomeString = some('Hello, world!');
+	 * const maybeSomeLength = maybeSomeString.map((value) => value.length);
```

Comment:
This example doesn't demonstrate `mapNoneInto`.

Reply:
Resolved in 04f23f4.